pipelines/data-generation/README.md

```diff
@@ -0,0 +1 @@
+# single-node
```

Comment:
can we add a description of what the pipeline does? A couple of high-level statements for each stage would suffice, along with a brief description on how to provide credentials (i.e. if I have an account on CloudLab, what do I do in order to make this work on my account?)